src/frequenz/client/common/pagination/pagination.py

```diff
@@ -0,0 +1,100 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Module to define the pagination used with the common client."""
+
+from __future__ import annotations  # required for constructor type hinting
+
+import logging
+from dataclasses import dataclass
+
+# pylint: disable=import-error, no-name-in-module
+from frequenz.api.common.v1.pagination_info_pb2 import (
+    PaginationInfo as PBPaginationInfo,
+)
+
+# pylint: disable=import-error, no-name-in-module
+from frequenz.api.common.v1.pagination_params_pb2 import (
+    PaginationParams as PBPaginationParams,
+)
```

Comment:
`# pylint: disable=import-error, no-name-in-module` in its own line disables that check from the point it appears until **the end of the file**. Also, I think on `no-name-in-module` is necessary. So you should do this instead:

```suggestion
# pylint: disable=no-name-in-module
from frequenz.api.common.v1.pagination_info_pb2 import (
    PaginationInfo as PBPaginationInfo,
)
from frequenz.api.common.v1.pagination_params_pb2 import (
    PaginationParams as PBPaginationParams,
)
# pylint: enable=no-name-in-module
```